addons/sale_management/controllers/portal.py

```diff
@@ -7,10 +7,10 @@
 from odoo.tools import formatLang
 from odoo.exceptions import AccessError, MissingError
 from odoo.http import request
-from odoo.addons.sale.controllers.portal import CustomerPortal
+from odoo.addons.sale.controllers.portal import CustomerPortal as CP
```

Comment:
![fbi](https://user-images.githubusercontent.com/7571158/96109957-86620a00-0edf-11eb-895a-7d3aca15e8b2.jpeg)

Reply:
stands for Cheese Pizza of course...

Reply:
![pizza](https://i1.wp.com/jfkfacts.org/wp-content/uploads/2018/08/Screen-Shot-2018-08-06-at-3.17.49-PM.png?ssl=1)

Reply:
didn't find cheese pizza but did find mayan "calander" :thinking:

Reply:
typo, they meant colander
![mayan_colander](https://user-images.githubusercontent.com/7571158/96217524-0f7e4d00-0f83-11eb-9627-c664e235c76e.jpeg)